server/services/locationService.test.ts

```diff
@@ -0,0 +1,92 @@
+import serviceCreator from './locationService'
+
+const elite2ClientBuilder = jest.fn()
+import incidentClient = require('../data/incidentClient')
+const token = 'token'
+
+const elite2Client = {
+  getActiveAgenciesByType: jest.fn(),
+  getPrisonById: jest.fn(),
+}
+
+incidentClient.updateAgencyId = jest.fn()
+
+let locationService
+
+beforeEach(() => {
+  elite2ClientBuilder.mockReturnValue(elite2Client)
+  locationService = serviceCreator({ elite2ClientBuilder })
+})
+
+afterEach(() => {
+  elite2Client.getActiveAgenciesByType.mockReset()
+  elite2Client.getPrisonById.mockReset()
+})
+
+describe('locationService', () => {
+  describe('getActiveAgenciesByType', () => {
+    it('should return no prison', async () => {
```

Comment:
`one prison` ? You don't strictly need this test- there isn't any particular logic around handling one prison

Reply:
done